README.md

```diff
@@ -6,16 +6,23 @@ The ELMU website
 
 ### Linting commands
 
-~~~
+~~~sh
 gulp lint // Will run eslint
 gulp fix  // Will run eslint and fix any linting rules that are easily solvable.
           // ATTENTION: commit your work before running this command.
 ~~~
 
-### Running locally
+### Tunnelling
 
-~~~
-gulp // Will start up mongodb, minio and maildev containers and run the project
```

Comment:
I think this would be good to have here too 🤔

Reply:
Yeah, this was a sacrifice to make ELMU look exactly like OMA except for when there was an actual difference. Honestly, I thought it would be not such a surprise what running `gulp` does (given that we for example never documented anywhere what `gulp faviconCheckUpdate` does). Maybe we should create an explicit ticket for documenting the development tasks (but then all of them)? For me, it would be enough to not document it and read it in the gulpfile itself.